.github/workflows/publish-pr-preview.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           surge-token: ${{ secrets.SURGE_TOKEN_DOC }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
           doc-site-branch: ${{ github.head_ref }}
-          build-preview-command: ./build-preview.bash --single-branch-per-repo --ignore-error true
+          build-preview-command: ./build-preview.bash --single-branch-per-repo
```

Comment:
Contrarily to the POC, we cannot ignore error as we use Antora 3.1 here. We keep the xref validation in the "build-pr-site" action